CMakeModules/EosioTesterBuild.cmake.in

```diff
@@ -60,7 +60,7 @@ else()
    find_library(libsecp256k1 secp256k1 @CMAKE_BINARY_DIR@/libraries/fc/secp256k1 NO_DEFAULT_PATH)
 endif()
 
-find_library(libff ff @CMAKE_BINARY_DIR@/libraries/fc/libraries/ff NO_DEFAULT_PATH)
+find_library(libff ff @CMAKE_BINARY_DIR@/libraries/fc/libraries/ff/libff NO_DEFAULT_PATH)
```

Comment:
does the other tester cmake need to be changed?

Reply:
also, #426

Reply:
Not if the 'dev' component is installed to the typical /usr/local.  I am updating the README for current install instructions to make this work.